tasks/android.py

```diff
@@ -65,12 +65,8 @@ def build(ctx, rebuild=False, race=False, build_include=None, build_exclude=None
     # Generating go source from templates by running go generate on ./pkg/status
     generate(ctx)
 
-    # remove all tags that are only available on debian distributions
-    distname = distro.id().lower()
-    if distname not in REDHAT_DEBIAN_SUSE_DIST:
-        for ex in REDHAT_DEBIAN_SUSE_ONLY_TAGS:
-            if ex not in build_exclude:
-                build_exclude.append(ex)
+    # remove all tags that are not available for current distro
+    build_exclude.extend(get_distro_exclude_tags())
```

Comment:
Seems `build_include` and `build_exclude` are not used, should remove the related code ?

Reply:
Go for it. This is the Android job which probably started like a copy&paste from the Agent job. Distros don't make sense anyway in this context.

Reply:
@albertvaka Sure, done.